CONTRIBUTING.md

```diff
@@ -41,16 +45,15 @@ Been here before? Already know what you're looking for in this guide? Jump to th
 ## Joining the community
 
 The Turing Way is a community-oriented and -led project.
-We therefore require that all contributions **adhere to our [Code of Conduct](CODE_OF_CONDUCT.md)**.
+We therefore require that all our members and their contributions **adhere to our [Code of Conduct](CODE_OF_CONDUCT.md)**. Please familiarize yourself with the [Code of Conduct](CODE_OF_CONDUCT.md) that lists the expected behaviours and our reporting guidelines.
 
 ## Inclusivity
 
-This project aims to be inclusive to people from all walks of life and to all research fields.
-This should be taken into account in contributions.
+This should be taken into account in contributions that the Turing Way aims to be inclusive to people from all walks of life and to all research fields.
```

Comment:
```suggestion
_The Turing Way_ aims to be inclusive to people from all walks of life and to all research fields.
These intentions are manifest in the contributions that we make to the project.
```

I felt like your original version was burying the lede of what the Turing Way is aiming to do. But my second sentence is pretty jargony....feel free to adjust.